app/views/comment_mailer/comment_reply_notification.text.erb

```diff
@@ -1,12 +1,5 @@
 <% content_for :message do %>
-<% responder = @comment.comment_owner ? 
-    (
-      @comment.by_anonymous_creator? ?
-      "Anonymous Creator" : 
-      @comment.comment_owner_name + " (" + polymorphic_url(@comment.comment_owner, :only_path => false) + ")"
-    ) :
-    @comment.comment_owner_name %>
-<%= responder %> replied to your comment on <% if @comment.ultimate_parent.is_a?(Tag) then %> the tag <%= @comment.ultimate_parent.commentable_name %> (<%= url_for(:controller => :tags, :action => :show, :id => @comment.ultimate_parent.to_param, :only_path => false) %>)<% else %>"<%= @comment.ultimate_parent.commentable_name.html_safe %>" (<%= polymorphic_url(@comment.ultimate_parent, :only_path => false) %>)<% end %>:
+<%= commenter_pseud_or_name_text(@comment) %> replied to your comment on <% if @comment.ultimate_parent.is_a?(Tag) then %> the tag <%= @comment.ultimate_parent.commentable_name %> (<%= url_for(:controller => :tags, :action => :show, :id => @comment.ultimate_parent.to_param, :only_path => false) %>)<% else %>"<%= @comment.ultimate_parent.commentable_name.html_safe %>" (<%= polymorphic_url(@comment.ultimate_parent, :only_path => false) %>)<% end %>:
```

Comment:
We should probably update the hash rockets to Ruby 1.9 hash syntax, e.g. `only_path: false`